### DISCUSSION TITLE
google chrome requiring link preload

### DISCUSSION THREAD

**Discussion by vasconx** [vercel/next.js#16961] [Help] [answered]
does anyone know how to get out of this? I looked in the issues but nobody managed to solve it ...

![Screenshot from 2020-09-09 09-51-01 (1)](https://user-images.githubusercontent.com/48503371/92603785-1a5c0880-f286-11ea-8233-9347df0d830d.png)
![Screenshot from 2020-09-09 10-15-32](https://user-images.githubusercontent.com/48503371/92603804-2051e980-f286-11ea-9379-3dc36cb86ec4.png)

**Comment by vasconx** [comment]
crossorigin="anonymous" doesnt work about second print

**Comment by jamesmosier** [comment, marked as answer]
I would not recommend this (your sites perceived performance is more important than Lighthouse), but you could [disable link prefetching](https://nextjs.org/docs/api-reference/next/link) which would then not add that `<link>` to your page. By default Next will add these prefetch `<link>` tags for each one of your `next/link` components on a page. This speeds up navigation to the next page. Here is how you disable it:

```js
import Link from 'next/link'

<Link href="/your-page" prefetch={false}><a>My link</a></Link>
```

**Reply by vasconx** [reply]
Okay, I'm going to say something very stupid. But my "_document.js" was written "_documents.js", it is a subtle difference but I think next.js documentation should emphasize more (although I recognize my stupidity) because the final project runs the same and the small changes that are made to the file are given as if it didn't work. It's been about 2 weeks since I saw some accusing the mistake I said, I don't know their case, but I don't put my hand on the fire. (Again, I recognize my stupidity). But thanks jamess for the answer and the time!!!
